dask_match/tests/test_collection.py

```diff
@@ -139,12 +139,34 @@ def test_repr(ddf):
 
 
 def test_columns_traverse_filters(df, ddf):
-    result = optimize(ddf[ddf.x > 5].y)
+    result = optimize(ddf[ddf.x > 5].y, fuse=False)
     expected = ddf.y[ddf.x > 5]
 
     assert str(result) == str(expected)
 
 
+def test_optimize_fusion():
+    pdf = pd.DataFrame({"x": range(10), "y": range(10)})
+    ddf = from_pandas(pdf, npartitions=1)
```

Comment:
These have fixtures now btw I think

Reply:
Yeah, sorry. This is leftover from me debugging with one partition.